_config.yml

```diff
@@ -175,10 +175,10 @@ fetch-remote:
         src:
           - "docs/configuration.md"
 
-  - repo: "https://github.com/moby/buildkit"
+  - repo: "https://github.com/crazy-max/buildkit"
     default_branch: "master"
-    ref: "master"
+    ref: "docs-ref-move-frontmatter"
```

Comment:
use my fork while waiting for https://github.com/moby/buildkit/pull/2970 to be merged